test/unit/urltils.test.js

```diff
@@ -273,4 +275,91 @@ describe('NR URL utilities', function () {
       })
     })
   })
+
+  describe('obfuscates path by regex', function () {
+    let config
+    let path
+    let loggerStub
+
+    beforeEach(function () {
+      loggerStub = {
+        './util/logger': function () {
+          return { warn: sinon.spy() }
+        },
+        'child': function () {
+          return {
+            logger: {
+              warn: sinon.spy()
+            }
+          }
+        },
+        'warn': sinon.spy()
+      }
+      proxyquire('../../lib/util/urltils', { '../logger': loggerStub })
```

Comment:
@bizob2828 Are you able to look into it and see if you can stub the `logger`, I tried various combinations but can't make it work. Joys of working with `proxyquire` and alike 😿

Reply:
yea i can comment